### ISSUE TITLE
[Problem/Bug]: NVDA does not read Find popup results correctly

### ISSUE THREAD

**Issue by pushkin-** [MicrosoftEdge/WebView2Feedback#5216]
### What happened?

NVDA does not read Find popup results properly.

If I go to a site like this: https://learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes/?tabs=dotnetcsharp

start NVDA, and search for `Prerelease`, NVDA reads "No Results" even when I see 28 results highlighted.
Sometimes, for other searches in my application, it will read "0 slash 1, 1 slash 1, 1 slash 2, 1 slash 2"

For example, if you search "a" on that Microsoft site above, it will read out that there's 998 results

### Importance

Moderate. My app's user experience is affected, but still usable.

### Runtime Channel

Stable release (WebView2 Runtime)

### Runtime Version

135.0.3179.85

### SDK Version

1.0.3116-prerelease

### Framework

Winforms

### Operating System

Windows 11

### OS Version

24H2 26100.3775

### Repro steps

see above

### Repros in Edge Browser

Yes, issue can be reproduced in the corresponding Edge version

### Regression

No, this never worked
